packages/shared/tailwind.config.ts

```diff
@@ -148,6 +148,7 @@ export default {
       6: '0.375rem',
       8: '0.5rem',
       10: '0.625rem',
+      11: '0.6875rem',
```

Comment:
Needed for inner border, else @omBratteng would get angry

Reply:
😂